event_project/models/event_event.py

```diff
@@ -0,0 +1,81 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2014 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
+#                 Endika Iglesias <endikaig@antiun.com>
+#                 Antonio Espinosa <antonioea@antiun.com>
+#                 Javier Iniesta <javieria@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    project_template = fields.Many2one(
+        comodel_name='project.project', string='Template project',
+        domain="[('state', '=', 'template')]")
+    project = fields.Many2one(
+        comodel_name='project.project', string='Related project',
+        readonly=True)
+
+    @api.model
+    def create(self, vals):
+        event = super(EventEvent, self).create(vals)
+        if (event.project_template and not event.project):
+            project_obj = self.env['project.project']
+            result = event.project_template.duplicate_template()
```

Comment:
That line and the assignation to the event. Take in note you have made in the create the tasks after the super call, and in write before. Do both in the same place.